app/src/main.ts

```diff
@@ -21,12 +20,32 @@ if (import.meta.env.VITE_FAV_ICON) {
     }
 }
 
+async function reportError(error: any, message?: string) {
+    if (import.meta.env.PROD) {
+        try {
+            const Sentry = await import("@sentry/vue");
+            if (message) {
+                Sentry.captureMessage(message);
+            }
+            Sentry.captureException(error);
+        } catch (e) {
+            console.error("Failed to load Sentry:", e);
+        }
+    }
+}
```

Comment:
Only import once, and keep a reference to the imported library
```suggestion
function reportError(error: any, message?: string) {
    if (import.meta.env.PROD) {
        if (!Sentry) {
               console.log("Sentry not loaded yet");
               return;
            }
            if (message) {
                Sentry.captureMessage(message);
            }
            Sentry.captureException(error);
    }
}
```